build/deploy/Dockerfile

```diff
@@ -3,11 +3,7 @@ FROM debian:jessie
 MAINTAINER Tobias Schottdorf <tobias.schottdorf@gmail.com>
 
 RUN mkdir -p /cockroach
-ADD cockroach /cockroach/
-ADD cockroach.sh /cockroach/
-ADD test.sh /cockroach/
-ADD ./resources /cockroach/resources/
-ADD ./ui /cockroach/ui
+ADD cockroach.sh test.sh build/cockroach build/resources build/ui /cockroach/
```

Comment:
Apparently COPY is preferred except when you need the additional ADD semantics. See https://docs.docker.com/articles/dockerfile_best-practices/#add-or-copy. I've been meaning to make this change to the other Dockerfile as well.